templates/zerver/integrations/github-actions.md

```diff
@@ -1,8 +1,27 @@
-It's easy to send Zulip messages from GitHub Actions! This is useful:
+# Zulip GitHub Actions integration
 
-  * If you want to report the success or failure of an Action to Zulip.
-  * If you want to send output from your actions to Zulip.
+It's easy to send Zulip messages from GitHub Actions!
 
-Instructions are available [here](https://github.com/zulip/github-actions-zulip#readme).
+{start_tabs}
 
-See also the [GitHub integration](/integrations/doc/github).
+1. {!create-a-generic-bot.md!}
+   Note down the bot's email address and API key.
+
+1. Integrate the `zulip/github-actions-zulip/send-message@v1` action into
+   your GitHub Actions workflow file, to send Zulip notifications
+   whenever your workflow runs.
```

Comment:
This should be a link -- otherwise, as a reader, I have no idea how to follow this step. https://github.com/zulip/github-actions-zulip/blob/main/send-message/README.md seems appropriate?

Reply:
What does "integrate" mean? Should it just be "Add the `zulip/github-actions-zulip/send-message@v1` action to your GitHub Actions workflow file..."

Reply:
I think these two steps are actually in the wrong order? It should basically be (not it these words):

1. Make a bot.
2. Define your action, using the bot's info, as per README.
3. Add the action to your workflow.